server/routes/organizations/employees/employee-controller.js

```diff
@@ -1,29 +1,79 @@
+var models = require('../../../db/models');
```

Comment:
thoughts on using app.get('models') here? i remember ryan mentioning it at some point when we were discussing the db - it will prevent errors caused by 
https://nodejs.org/api/modules.html#modules_caching (see module caching caveats).  i think this blog post also covers something similar: 
http://www.redotheweb.com/2013/02/20/sequelize-the-javascript-orm-in-practice.html

have you seen something to the contrary?